docs/user-guide/map-templates.md

```diff
@@ -0,0 +1,66 @@
+# Map Templates
+*******************
+
+The **Map Templates** is an extension added to the context to browse the map template like the *WMC templates* and the *MapStore JSON*. The <img src="../img/button/map-templates-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the available templates. 
```

Comment:
This extension allows to browse map templates in a MapStore's viewer. Supported Map Templates formats in MapStore are [WMC](https://mapstore.readthedocs.io/en/latest/developer-guide/maps-configuration/#web-map-context) and [MapStore's native JSON](https://mapstore.readthedocs.io/en/latest/developer-guide/maps-configuration/#map-options). The <img src="../img/button/map-templates-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the available templates.

>The section corresponding to this [link ](https://mapstore.readthedocs.io/en/latest/user-guide/import/#import-map-context-files)must be updated to document also the WMC format for import and export and we must have a dedicated section for the Export functionality